erpnext/erpnext_integrations/lenco_subscription.py

```
@@ -0,0 +1,81 @@
+@dataclass
+class LencoSubscription:
+    """Handle subscription payments via Lenco."""
+
+    plan: str
+    user_count: int = 1
+
+    BASE_PLAN_PRICES = {
+        "monthly": 2500,
+        "quarterly": 7000,
+        "yearly": 35000,
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)